### DISCUSSION TITLE
Example for balance query

### DISCUSSION THREAD

**Discussion by dezur** [Bad-Pop/JCoinbase#14] [Q&A]
Hi! I am new to Java. Could anyone please provide an example code for getting an account balance for all crypto? I tried
`client.account().getAccountsPageAsJava().get().getAccountsAsJava();` but it only shows a few.

**Comment by Bad-Pop** [comment]
Hi @dezur !
Thanks for your message !

If you have multiple cryptoassets in your account, Coinbase will paginate the returned responses. This avoids long processing times and too much data flowing at once.
So, if you want to get all the accounts you have, you will have to make a call for each available page.

Have a great day, and I hope you enjoy JCoinbase

**Reply by dezur** [reply]
I get a NoNextPageException when I try to use getNextAccountsPageAsJava(). Sorry if it's a noob question but can you show me how pagination works?

**Reply by Bad-Pop** [reply]
> I get a NoNextPageException when I try to use getNextAccountsPageAsJava(). Sorry if it's a noob question but can you show me how pagination works?

Can you show me the response from Coinbase when you first call? **(be careful to hide sensitive data before publishing)**

EDIT: And with the logs it will also help
